src/cmake-tools.ts

```diff
@@ -909,6 +963,32 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    */
   async clean(): Promise<number> { return this.build('clean'); }
 
+  /**
+   * Implementation of `cmake.openConfiguration`
+   */
+  async openConfiguration(): Promise<number> {
+    if (!this._webview) {
+      const drv = await this.getCMakeDriverInstance();
+      if (!drv) {
+        vscode.window.showErrorMessage('No CMakeCache.txt file has been found. Please configure project first!');
```

Comment:
User-facing messages need to be translated into the appropriate language.  Please call `localize` on your message here and give it an id.  Our translation team will take care of the rest.